extensions.txt

```diff
@@ -0,0 +1,61 @@
+AWE_Info
```

Comment:
What is this file for? We should at least include a comment as to why it exists or note its purpose in the documentation.